content/docs/alerting/rules.md

```diff
@@ -91,7 +91,7 @@ Prometheus's alerting rules are good at figuring what is broken *right now*,
 but they are not a fully-fledged notification solution. Another layer is needed
 to add summarization, notification rate limiting, silencing and alert
 dependencies on top of the simple alert definitions. In Prometheus's ecosystem,
-the [Alertmanager](../alertmanager) takes on this
+the [Alertmanager](/docs/alertmanager) takes on this
```

Comment:
Should this link to `/docs/alerting`? Because `/docs/alertmanager` doesn't exist yet I think, at least not on your demo site.